ui/src/app/components/help-sidebar.tsx

```diff
@@ -492,7 +492,7 @@ export const HelpSidebar = fp.flow(
     sidebarContainerStyles(activeIcon, notebookStyles) {
       const sidebarContainerStyle = {
         ...styles.sidebarContainer,
-        width: `calc(${this.sidebarWidth}rem + 55px)`
+        width: `calc(${this.sidebarWidth}rem + 70px)`
```

Comment:
nit: can you add a comment do indicate what 70 corresponds to? Is this just the width of the sidebar?